.gitignore

```diff
@@ -41,4 +42,4 @@ testem.log
 
 # schematics
 /src/lib/schematics/**/*.js
-/src/lib/schematics/**/*.map
+/src/lib/schematics/**/*.map
```

Comment:
Can you configure your IDE to always add a newline at the end of files? Looks like it removed the newline here.